metallb/base/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: metallb
```

Comment:
Here you're using `metallb` namespace, while here it originally wants to deploy to `metallb-system`.
https://github.com/operate-first/apps/pull/405/files#diff-cf2e704bae45f65353f3bcc6dceaa69b150875d472fd9a5322253c00006c550bR8
and you're defining `metallb-system` namespace here:
https://github.com/operate-first/apps/pull/405/files#diff-0a4f86bc5dd47a741254b8a22c8c8e58196d4858a93a75c95c470d4f1958eea7R4